site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
+- [`chrome.contentSettings`](/docs/extensions/reference/contentSettings/)
+- [`chrome.debugger`](/docs/extensions/reference/debugger)
+- [`chrome.downloads`](/docs/extensions/reference/downloads)
+- [`chrome.enterprise.hardwarePlatform`](/docs/extensions/reference/enterprise_hardwarePlatform)
+- [`chrome.fontSettings`](/docs/extensions/reference/fontSettings)
+- [`chrome.history`](/docs/extensions/reference/history)
+- [`chrome.instanceID`](/docs/extensions/reference/instanceID)
+- [`chrome.permissions`](/docs/extensions/reference/permissions)
+- [`chrome.processes`](/docs/extensions/reference/processes)
+- [`chrome.search`](/docs/extensions/reference/search)
+- [`chrome.sessions`](/docs/extensions/reference/sessions)
+- [`chrome.signedInDevices`](/docs/extensions/reference/signedInDevices)
+- [`chrome.topSites`](/docs/extensions/reference/topSites)
+
+Also, APIs that use the [`ChromeSetting`](/docs/extensions/reference/types/#type-ChromeSetting)
+prototype now also support promises. The following APIs are affected by this change.
+
+- [`chrome.privacy`](/docs/extensions/reference/privacy/)
+- [`chrome.accessibilityFeatures`](/docs/extensions/reference/accessibilityFeatures/)
+- [`chrome.proxy`](/docs/extensions/reference/proxy/)
+
+{% endDetails %}
+
+{% Details %}
+{% DetailsSummary %}
+Chrome OS APIs
+{% endDetailsSummary %}
+
+- [`chrome.certificateProvider`](/docs/extensions/reference/certificateProvider)
+- [`chrome.documentScan`](/docs/extensions/reference/documentScan)
+- [`chrome.enterprise.deviceAttributes`](/docs/extensions/reference/enterprise_deviceAttributes)
+- [`chrome.enterprise.networkingAttributes`](/docs/extensions/reference/enterprise_networkingAttributes)
+- [`chrome.fileBrowserHandler`](/docs/extensions/reference/fileBrowserHandler)
+- [`chrome.fileSystemProvider`](/docs/extensions/reference/fileSystemProvider)
+- [`chrome.loginState`](/docs/extensions/reference/loginState)
+- [`chrome.printingMetrics`](/docs/extensions/reference/printingMetrics)
+- [`chrome.wallpaper`](/docs/extensions/reference/wallpaper)
+
+{% endDetails %}
+
+### Chrome 96: `declarativeNetRequestWithHostAccess` permission
+
+The new `declarativeNetRequestWithHostAccess` permission allows existing Manifest V2 extensions that
+leverage `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
+[`chrome.declarativeNetRequest`](/docs/extensions/reference/declarativeNetRequest/) API without
+requiring the end user to approve a new permissions grant.
+
+### 2021-10-27: Review violation troubleshooting updates {: #2021-10-27-reivew-troubleshooting }
 
 The [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/) page has been
 updated to provide developers with more detailed guidance for common reasons for rejection.
 
-### Manifest V2 support timeline
+### Chrome 95: Inject scripts directly into pages {: #m95-page-script-injection }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting) API's
```

Comment:
>The `chrome.scripting` API's

Please be specific. What is an API called in the Extensions universe? Class? Object? Interface?

Reply:
Please look at this throughout. This isn't just pickiness. You're missing an opportunity to provide information readers who are new to Extensions.